pipelines/pipeline_katello_33_to_34_upgrade.yml

```diff
@@ -0,0 +1,113 @@
+- hosts: localhost
+  vars:
+    forklift_name: pipeline-katello-3.3-3.4
+    forklift_boxes:
+      pipeline-katello-3.3-3.4-centos7:
+        box: centos7
+        memory: 4680
+      pipeline-proxy-3.3-3.4-centos7:
+        box: centos7
+        memory: 3072
+  roles:
+    - forklift
+
+- hosts:
+  - pipeline-katello-3.3-3.4-centos7
+  - pipeline-proxy-3.3-3.4-centos7
+  become: yes
+  vars:
+    puppet_repositories_version: 4
+    katello_repositories_version: 3.3
+    foreman_repositories_version: 1.14
+    foreman_installer_skip_installer: true
+  roles:
+    - selinux
+    - etc_hosts
+    - epel_repositories
+    - puppet_repositories
+    - foreman_repositories
+    - katello_repositories
+    - disable_firewall
+    - haveged
+    - update_os_packages
+
+- hosts: pipeline-katello-3.3-3.4-centos7
+  become: yes
+  vars:
+    foreman_installer_scenario: katello
+    foreman_installer_options_internal_use_only:
+      - "--foreman-admin-password {{ foreman_installer_admin_password }}"
+      - "--disable-system-checks"
+    foreman_installer_additional_packages:
+      - katello
+  roles:
+    - foreman_installer
+    - hammer_config
```

Comment:
In plain foreman there's an option to enable the CLI which should set it up for you. Won't that be sufficient or is it not available in the katello scenario?